src/ScrollTargetValidator.hpp

```diff
@@ -0,0 +1,32 @@
+/************************************************************************/
+/*  Xania (M)ulti(U)ser(D)ungeon server source code                     */
+/*  (C) 2021 Xania Development Team                                     */
+/*  See the header to file: merc.h for original code copyrights         */
+/************************************************************************/
+#pragma once
+
+#include "ObjectIndexValidator.hpp"
+#include "SkillTables.hpp"
+
+#include <map>
+#include <optional>
+
+class ObjectIndex;
+enum class Target;
```

Comment:
Ah! it's `Target` is only an `enum` ... very much weakens my argument above. Sorry (leaving the comments there for discussion only)